sedai/README.md

```diff
@@ -1,59 +1,42 @@
 
-From preventing downtime and managing SLOs to analyzing code releases and lowering cloud costs, Sedai is the autonomous platform for your SRE and DevOps teams. As a Sedai customer, you can directly integrate with Datadog to receive select notifications in your dashboards.
+Enable this integration to receive notifications in Datadog about actions that Sedai autonomously executes in your production environments.
 
-**Unlock the full potential of your microservices:**
-* Achieve 99.99% or better application availability
-* Autonomously reduce resource latency by 75%
-* Reduce cloud costs by 30% through autonomous optimization
-* Increase team productivity and agility by 3x
+### How it Works
 
-### Core features
+* **Agentless:** Seamlessly connects to your cloud accounts and automatically discovers and understands production environments.
 
-* **Easy setup:** Sedai's agentless approach integrates with your cloud accounts and observability platforms so you can get up and running in minutes.
-* **Configuration-free:** By inferring your topology and metrics, Sedai leverages them to detect signals based on observed performance.
-* **Autonomous Availability:** Prevent issues and ensure the highest levels of availability with autonomous actions.
-* **Release Intelligence:** Validate release performance in production in real-time with smart scorecards that assess the quality of your deployments and analyze deviations in latency and errors.
-* **Resource Optimization:** Improve customer experience and lower cloud costs on the fly with dynamic allocation that manages your resources based on traffic needs.
-* **Smart SLOs:** In addition to automatically analyzing and defining targets, Sedai manages error budgets with autonomous actions that prevent errors, downtime, and SLO breaches.
+* **Configuration-free:** Easily connects to Datadog API and intelligently identifies, prioritizes and learns metric behavior.
 
-To learn more, see [Sedai Inc.][1]
+* **Proactive Actions:** Safely acts in production on your behalf to ensure that resources avoid availability issues and run optimally at all times.
 
 ## Setup
 
-After connecting your cloud provider, monitoring provider, and notification provider accounts, Sedai identifies your topology and metrics and starts to analyze performance.
-
-### Installation
-
-Existing Sedai customers can receive notifications about Sedai activity in Datadog. For more information, see the [Sedai Notifications][6].
+In Sedai:
 
-## Data Collected
-
-### Metrics
+* Navigate to Settings > Notifications > Add Integration > Datadog Icon 
+![DataDog Notification Integration](https://raw.githubusercontent.com/DataDog/integrations-extras/master/sedai/images/DataDog_Notification_Integration.png)
```

Comment:
```suggestion

![DataDog Notification Integration][3]
...
[3]: https://raw.githubusercontent.com/DataDog/integrations-extras/master/sedai/images/DataDog_Notification_Integration.png)
```

Insert a newline so that the images aren't inline with the previous line of text. Also, can we move the image URLs to the reference links section?

Reply:
Sure, done and PR is updated.